client/src/client/database_manager_client.py

```diff
@@ -17,67 +17,174 @@
 class DatabaseManager:
     """Clientside database manager."""
 
-    def __init__(self, name: str, password: str) -> None:
+    def __init__(
+        self, name: str, password: str, username: str, color: str
+    ) -> None:
         """Construct the clientside database manager."""
         self.log = logging.getLogger("cans-logger")
         self._db_name = str(name)
         self._db_pass = password
+        self.myself = Friend(id="myself", username=username, color=color)
```

Comment:
Please add "myself" and "system" in the upper layer (Client class) using the API exposed by the DatabaseManager class

Reply:
I thought implementing the special cases of users directly in DatabaseManager would be a better idea than doing it in the Client as I could insert them during database initialization and filter them more cleanly while fetching friends. I will comply with your request.

Reply:
I second the idea of special users being defined in client, they have nothing to do with the database 
Filtering them is not that important, just a cool feature :)